### ISSUE TITLE
lgsvl_bridge cannot work?

### ISSUE THREAD

**Issue by hanyh1191** [lgsvl/simulator#2047]
Hello, Everything are OK before today.
Today I start "lgsvl_bridge" from bash command, then show "[INFO] [lgsvl-bridge]: Listening on port 9090", this all OK. When I click "Run Simulation" from WebUI and start simulation, Then ERROR "[ERROR] [lgsvl-bridge]: rcl_publisher_init failed: 103" Just like the picture, all other config or source "setup.bash" command normal. Why? Thanks.
![2022-04-03 14-10-09 的屏幕截图](https://user-images.githubusercontent.com/67039858/161414469-064ec395-2034-4f77-81f5-29876f20ce53.png)

**Comment by hanyh1191** [comment]
yeah, how to say, I found my AD stack not care this ERROR, and still work OK though I don't know why cause error.

**Comment by carTloyal123** [comment]
You probably did not add a topic name to one of your sensors when setting up the sensor configuration. Check each sensor you have and make sure it has a unique topic name.
